### PR TITLE
Restructure Topbeat

### PR DESCRIPTION
**Pull request by ruflin**
* Use new simplified run methods provided by libbeat
* Update libbeat to most recent version
* Move brun test into tests folder
* Move all topbeat specific code into beat package

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,10 +1,8 @@
 package main
 
 import (
-	"os"
-
 	"github.com/elastic/libbeat/beat"
-	"github.com/elastic/libbeat/logp"
+	topbeat "github.com/elastic/topbeat/beat"
```
Comment (urso) [inline main.go]: maybe have a newline between imports to stress they are from different packages?

**Comment by urso** [comment]
LGTM.

File: main.go (hunk 1 above)
Comment (monicasarbu) [inline main.go]: that would be nice to have

**Comment by monicasarbu** [comment]
LGTM

File: main.go (hunk 1 above)
Comment (ruflin) [inline main.go]: Done. I put the local important on top as we have it in other packages.

**Comment by tsg** [comment]
Please also update the Developer guide from libbeat!

**Comment by ruflin** [comment]
@tsg Good point. Opened an issue here: https://github.com/elastic/libbeat/issues/330

**Pull request merged**